### PR TITLE
Update SoundEngine's viewpoint after regeneration

### PR DESCRIPTION
**Pull request by ad-daniel**
**Description**
Ditto.

### CODE REVIEW

File: src/webots/sound/WbSoundEngine.cpp [hunk 1]
```diff
@@ -128,20 +127,24 @@ bool WbSoundEngine::openAL() {
 void WbSoundEngine::setWorld(WbWorld *world) {
   if (world) {
     gWorld = world;
-    gViewpoint = world->viewpoint();
-    if (gViewpoint)
-      QObject::connect(gViewpoint, &WbViewpoint::cameraParametersChanged, &WbSoundEngine::updateListener);
+    if (gWorld && gWorld->viewpoint())
```
Comment (omichel) [inline src/webots/sound/WbSoundEngine.cpp:130]: That should be sufficient.
```suggestion
    if (gWorld)
```

**Review by omichel — CHANGES_REQUESTED** [review]

File: src/webots/sound/WbSoundEngine.cpp (hunk 1 above)
Comment (ad-daniel) [reply src/webots/sound/WbSoundEngine.cpp:130]: Indeed, doesn't appear to be necessary

**Review by omichel — APPROVED** [review]
Thank you.

**Comment by ad-daniel** [comment]
Ah right, didn't notice `world` was already checked above

**Review by omichel — APPROVED** [review]

**Pull request merged**